other-docs/guides/getting-help-with-altis.md

```diff
@@ -0,0 +1,50 @@
+# Getting help with Altis.
+
+If you need help, this guide will show you how to most effectively get you the help you need.
+
+While we support a few methods, the primary method and the route you should choose, is the Altis Dashboard Support button.
```

Comment:
```suggestion
## Altis Dashboard

While we support a few methods, the primary method and the route you should choose, is the Altis Dashboard Support button.
```

If we add a second level heading here it'll round off the methods of getting support more neatly. We should also make the ticket type headings into 3rd level headings to get a sensible hierarchy.